emark/backends.py

```diff
@@ -76,19 +94,26 @@ def write_message(self, message):
         return msg
 
 
-class TrackingConsoleEmailBackend(TrackingEmailBackendMixin, ConsoleEmailBackend):
+class ConsoleEmailBackend(
+    RenderEmailBackendMixin, ConsoleEmailBackendMixin, _EmailBackend
+):
+    """Like the console email backend but only with the plain text body."""
+
+
+class SMTPEmailBackend(RenderEmailBackendMixin, _SMTPEmailBackend):
+    """SMTP email backend that renders messages before establishing an SMTP transport."""
+
+    pass
+
+
+class TrackingConsoleEmailBackend(
+    TrackingEmailBackendMixin, ConsoleEmailBackendMixin, _EmailBackend
+):
     """Like the console email backend but with click and open tracking."""
 
     def write_message(self, message):
-        for recipient in message.recipients():
-            clone = copy.copy(message)
-            clone.to = [recipient]
-            clone.cc = []
-            clone.bcc = []
-            # enable tracking
-            clone._tracking_uuid = uuid.uuid4()
-            msg = super().write_message(clone)
-            self._track_message_clone(clone, msg)
+        self._track_message(message)
+        return super().write_message(message)
```

Comment:
I feel like that the actual action (so `write_message` logic) should be done first, then we do the tracking part.

```suggestion
        super().write_message(message)
        self._track_message(message)
```